pages/common/assets/sass/style.scss

```diff
@@ -316,3 +316,19 @@ p.spoc {
     }
   }
 }
+
+.pil-actions {
+  section {
+    border-top: 1px solid govuk-colour('grey-2');
+    padding: $govuk-gutter 0;
+  }
+}
+
+details {
+  margin-bottom: $govuk-gutter;
+
+  summary {
+    color: #005ea5;
```

Comment:
is this `$govuk-link-colour`? if so could we use the variable, or if it's a new thing could we define it at the top of the file